packages/api/amplify_api_dart/lib/src/graphql/factories/graphql_request_factory.dart

```diff
@@ -169,7 +169,8 @@ class GraphQLRequestFactory {
       case GraphQLRequestOperation.onCreate:
       case GraphQLRequestOperation.onUpdate:
       case GraphQLRequestOperation.onDelete:
-        // upperOutput and lowerOutput already '', done
+        upperOutput = '(\$filter: ModelSubscription${modelName}FilterInput)';
```

Comment:
question: In an older backend w no server-side subscription support, would this throw an error from AppSync when caller provides no filter (would not expect `ModelSubscriptionBlogFilterInput` to be present in transformed schema)? If so, would it be possible to leave this blank when no filter provided?